src/Hazelcast.Net/Exceptions/ClientNotAllowedInClusterException.cs

```diff
@@ -74,7 +74,9 @@ public ClientNotAllowedInClusterException(string message, Exception innerExcepti
         /// about the exception being thrown.</param>
         /// <param name="context">The <see cref="StreamingContext"/> that contains contextual information
         /// about the source or destination.</param>
+#pragma warning disable CA2229 // Implement serialization constructors
```

Comment:
it's complaining that the ctor should not be `private` if the class is unsealed (so potentially can have classes inheriting from it, which would need to use the ctor, so it should be `protected`) - but we don't plan to have classes inherit from that one, so better mark the class `sealed` and then the warning goes away

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/682/commits/146cc487e6723800e0609781507b11b70e1c1c41

Reply:
ok